VirtoCommerce.Platform.Web/Model/Asset/AssetListItem.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace VirtoCommerce.Platform.Web.Model.Asset
+{
+    public class AssetListItem
```

Comment:
This type should be defined in the Platform.Core project we must avoid of Web models usage